### ISSUE TITLE
[Accessibility]: getChartID not being generated/available causing accessibility issues

### ISSUE THREAD

**Issue by huineng** [carbon-design-system/carbon-charts#1716]
### Name

Guy Huinen

### Are you an IBM employee?

- [X] Yes

### Environment

Automated testing tool and ruleset

### What happened? What did you expect to happen instead?

IBM Equal Access Accessibility Checker revealed several issues, but they are mainly caused by the absent of the ChartId

### What WCAG 2.1 checkpoint does the issue violate?

element_id_unique

### Version

 "@carbon/charts": "^1.13.15",

![image](https://github.com/carbon-design-system/carbon-charts/assets/3482445/8e0178a0-ed9c-4bce-a1a9-f02501ac96b7)

### Data & options used

```js
This is not really an issue if there's only one chart on the page
But it is when you have 2 or more because of the absence of the chartID, many elements are named the same
```

### Relevant log output

```js
you can see from the picture above in inspector that the ChartId is empty when rendering
```

### What priority level would this issue be in your opinion?

P0

**Comment by huineng** [comment]
I see your latest release solved this issue.
Thanks, i will close this ticket
